README.md

```diff
@@ -4,13 +4,31 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  - We may have many constants which we dont take into account in asymptomatic analysis
+  - Ignoring ordering of terms. We may have one million elements to sort, however we come to the same asymptomatic conclusion with the same algorithm that we would if we were sorting ten elements
```

Comment:
Don't understand this. The input size is part of the expression, so we're not ignoring it.